control/controlclient/auto.go

```diff
@@ -590,7 +590,7 @@ func (c *Auto) sendStatus(who string, err error, url string, nm *netmap.NetworkM
 	}
 	if nm != nil && loggedIn && synced {
 		pp := c.direct.GetPersist()
-		p = &pp
+		p = pp.AsStruct()
```

Comment:
Should this be passing down the `PersistView` instead? I assume we don't expect `statusFunc` to mutate things.